src/pages/docs/src/codegen/gen-bundlers.js

```diff
@@ -0,0 +1,28 @@
+const { generated, getName, getBundlers } = require('./paths');
+const { render } = require('./markdown');
+const { generateStandardPage } = require('./pages');
+
+function getRoute(name) {
+  return (name && `/bundlers/${name}`) || '';
```

Comment:
Can we have a `/tooling/...` path here - like for the other sections in tooling?

Reply:
adapted